content/en/docs/concepts/containers/images.md

```diff
@@ -380,6 +383,42 @@ Kubernetes supports specifying container image registry keys on a Pod.
 `imagePullSecrets` must all be in the same namespace as the Pod. The referenced
 Secrets must be of type `kubernetes.io/dockercfg` or `kubernetes.io/dockerconfigjson`.
 
+#### Ensure Image Pull Credential Verification {#ensureimagepullcredentialverification}
+
+{{< feature-state feature_gate_name="KubeletEnsureSecretPulledImages" >}}
+
+If the `KubeletEnsureSecretPulledImages` feature gate is enabled, Kubernetes
+will validate image credentials for every image that requires credentials 
+to be pulled, even if that image is already present on the node. This validation
+ensures that images in a pod request which have not been successfully pulled with the
+provided credentials must re-pull the images from the registry.
+Additionally, image pulls that re-use the same credentials which previously resulted 
+in a successful image pull will not need to re-pull from the registry and are instead 
+validated locally without accessing the registry (provided the image is available locally).
+This is controlled by the`imagePullCredentialsVerificationPolicy` field in the
+[Kubelet configuration](/docs/reference/config-api/kubelet-config.v1beta1#ImagePullCredentialsVerificationPolicy).
+
+This configuration controls when image pull credentials must be verified if the
+image is already present on the node:
+
+ * `NeverVerify`: Mimics the behavior of having this feature gate disabled. If the image
+ is present locally, image pull credentials are not verified.
```

Comment:
We use sembr (https://sembr.org) style of line breaks.
Also, we recommend indenting all lines in a list item. I.e.

```suggestion
 * `NeverVerify`: Mimics the behavior of having this feature gate disabled.
   If the image is present locally, image pull credentials are not verified.
```